tests/template/elegant_test.py

```diff
@@ -27,7 +27,7 @@ def test_file_iterator():
     from sirepo.template.lattice import LatticeUtil
     from pykern.pkunit import pkeq
     data = _find_example('bunchComp - fourDipoleCSR')
-    v = LatticeUtil(data, _elegant()._SCHEMA).iterate_models(
+    v = LatticeUtil(data, _elegant().SCHEMA).iterate_models(
```

Comment:
This seems not right. Shouldn't it get the information from sim_data?

Reply:
Can you elaborate? Should it get the schema from sim_data (ex `_elegant()._SIM_DATA.schema()`)?

Reply:
yes, there shouldn't be a sharing of that variable unless it is necessary.

Reply:
Thanks b24452abc